cli/copy_test.go

```diff
@@ -239,6 +239,16 @@ func TestCopyRootfsImage(t *testing.T) {
 				os.Remove("input.txt")
 			},
 		},
+		{
+			initfunc: func(imgpath string) {
+				assert.Nil(t, ioutil.WriteFile("input.mender.txt", []byte("dummy-text"), 0755))
+			},
+			name: "copy file into a non-existing directory",
```

Comment:
Name tells me this should error, but `/etc/mender/` exists in the image.
Copy paste error? :thinking:

Reply:
Right. Thanks